packages/grafana-prometheus/package.json

```diff
@@ -0,0 +1,151 @@
+{
+  "author": "Grafana Labs",
+  "license": "Apache-2.0",
```

Comment:
I've not read all the code but it seems this package is pretty much the entire plugin minus the module.ts and plugin.json? All code in the Grafana codebase (except the NPM packages that we publish publicly) are AGPL-3.0. Are we sure we want to change this to Apache-2.0? 🤔

Reply:
Good point! We had not decided on the license yet. We know that there are some finer details to iron out with the legal team for how the custom vendors will use this code. If people are creating their own Prometheus plugins, we want to be in alignment with Grafana for the license.